roles/letsencrypt/tasks/convert_certificate.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: convert certificate to pfx format
+  shell: "openssl pkcs12 -inkey {{ private_key_path }} -in {{ fullchain_path }} -export -out {{ pfx_cert_path }}"
```

Comment:
Please use the ansible module instead of shell here: https://docs.ansible.com/ansible/latest/collections/community/crypto/openssl_pkcs12_module.html